src/parser/resource-parser.js

```diff
@@ -176,18 +173,16 @@ function parseAlertChannelSubscriptions (
 ) {
   const alertChannelSubscriptions = {}
 
-  if (resource.alertChannelSubscriptions) {
-    resource.alertChannelSubscriptions.forEach((subscription) => {
-      const { name, activated } = subscription
-      const logicalId = `${resourceLogicalId}/${name}`
+  resource.alertChannelSubscriptions.forEach((subscription) => {
```

Comment:
This can no longer be null?

Reply:
I moved the validation to the upper function `parseAlertChannelSubscriptionsTree` but when using schema at parser we always get a value (empty array as default).